docs/custom.css

```
@@ -0,0 +1,610 @@
+@import url("https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@400;500;600&display=swap");
```

Stylelint (17.9.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@400;500;600&display=swap"" (import-notation)

(import-notation)